contrib/kvm/README.md

```diff
@@ -0,0 +1,83 @@
+# KVM-based Ubuntu Environment of SOLVCON
+To automate Ubuntu installlation to run [SOLVCON](https://github.com/solvcon/solvcon).
+
+# Getting Started
+
+## Prerequisites
+An Ubuntu Xenial desktop with the following files/tools/scripts of the packages installed:
+- *libvirt.pc* (provided by *libvirt-dev*)
+- *Python.h* (provided by *libpython3.5-dev*)
```

Comment:
Is it possible to use Python 3.6 (on Xenial)?  SOLVCON minimal requirement is already 3.6

Reply:
Regarding Xenial/Python 3.6, it is certainly "yes". I am a little surprised because the basic nosetests work pretty well on Trusty. Maybe we could consider to add more tests to cover those python 3.6 specific feature.

Reply:
Travis CI runs with Python 3.6 on trusty.  I'll stick with the latest Python release.  But I can't follow Ubuntu version so closely.  Ubuntu releases too frequently and CI services are usually too slow to catch up.

For Ubuntu version, being xenial or trusty doesn't really matter.